src/components/faktum/faktum-land/FaktumLandReadOnly.tsx

```diff
@@ -8,13 +8,23 @@ import { IFaktumReadOnly } from "../Faktum";
 import { HelpText } from "../../HelpText";
 import styles from "../Faktum.module.css";
 import { PortableText } from "@portabletext/react";
+import { AlertText } from "../../alert-text/AlertText";
 
 export function FaktumLandReadOnly(props: IFaktumReadOnly<IQuizLandFaktum>) {
   const router = useRouter();
   const { faktum, showAllFaktumTexts } = props;
-  const { getFaktumTextById, getAppText } = useSanity();
+  const { getFaktumTextById, getAppText, getLandGruppeTextById } = useSanity();
   const faktumTexts = getFaktumTextById(faktum.beskrivendeId);
 
+  function getLandGruppeId(code: string) {
```

Comment:
Denne funksjonen er vel identisk med den som ligger i FaktumLand? Vi kan kanskje trekke den ut i en util fil?

Reply:
Ja, jeg tenkte på det.Eventuelt trekke den på toppen av `FaktumLand.tsx` og exporte den derfra. Skal se hva jeg tror er en god løsning der. 😊

Reply:
Jeg trodde ikke vi hadde en util-fil for faktum, men det hadde vi jaggu! La funksjonen der, slik at vi slipper dupliseringen 🎉